### DISCUSSION TITLE
Table Column Widths

### DISCUSSION THREAD

**Discussion by gregcowell** [beeware/toga#1618] [Q&A] [answered]
Is there currently a way to configure Table column widths ?

**Comment by freakboy3742** [comment, marked as answer]
At present, no.

We have precedent for specifying this sort of thing in SplitContainer (where it is possible to specify the % split between parts); there's also been some discussion about Table styling (#1478). However, neither of those are options that exist at present. Initial column width is the default as specified by the platform, based on table content and column titles.

**Reply by gregcowell** [reply]
Thanks. On the Windows/Winforms platform, the initial column width does not seem to be influenced by the column titles. The column width is smaller than the titles unless the titles are particularly small.
